### PR TITLE
Regenerate angular messages.xlf

### PR DESCRIPTION
**Pull request by shamoon**
Addresses #135 [comment](https://github.com/paperless-ngx/paperless-ngx/pull/135#issuecomment-1048970786)

### CODE REVIEW

**Comment by yschroeder** [comment]
You're right, I think I forgot that file. Did it translate to the same ids as the other files?

**Comment by shamoon** [comment]
It looks like it, not like I can check every one but jumping around, Ill make some comments

File: src-ui/messages.xlf [hunk 1]
```diff
@@ -1379,385 +1234,1010 @@
...
+        </context-group>
+      </trans-unit>
+      <trans-unit id="3100631071441658964" datatype="html">
+        <source>Title &amp; content</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/components/document-list/filter-editor/filter-editor.component.ts</context>
+          <context context-type="linenumber">89</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="1010505078885609376" datatype="html">
```
Comment (shamoon) [inline src-ui/messages.xlf]: https://github.com/paperless-ngx/paperless-ngx/blob/dbdd9b93f8f48f4a8e792418fa0f103b73e59d29/src-ui/src/locale/messages.de_DE.xlf#L1230

**Comment by yschroeder** [comment]
> It looks like it, not like I can check every one but jumping around, Ill make some comments

It seems to have changed the order though

**Comment by shamoon** [comment]
Well order shouldn't matter, it matches IDs.

https://github.com/paperless-ngx/paperless-ngx/blob/dbdd9b93f8f48f4a8e792418fa0f103b73e59d29/src-ui/src/locale/messages.de_DE.xlf#L2089 = https://github.com/paperless-ngx/paperless-ngx/blob/5941c383b6aa48863f4b01bf66ff21624281ddf9/src-ui/messages.xlf#L2311

https://github.com/paperless-ngx/paperless-ngx/blob/dbdd9b93f8f48f4a8e792418fa0f103b73e59d29/src-ui/src/locale/messages.de_DE.xlf#L1230 = https://github.com/paperless-ngx/paperless-ngx/blob/5941c383b6aa48863f4b01bf66ff21624281ddf9/src-ui/messages.xlf#L1508

**Review by yschroeder — APPROVED** [review]
LGTM. Can you document somewhere how to regenerate the xlf file correctly?

**Pull request merged**